<launch>
      <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
      <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>


    <node name="turtle1_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>
    <node name="turtle2_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle2" /> 
    </node>
    <node name="turtle3_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle3" />
    </node>
    <node name="turtle4_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle4" />
    </node>
    <node name="turtle5_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle5" />
    </node>
    <node name="turtle6_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle6" />
    </node>
    <node name="turtle7_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle7" />
    </node>
    <node name="turtle8_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle8" />
    </node>
    <node name="turtle9_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle9" />
    </node>
    <node name="turtle10_tf2_broadcaster" pkg="punto5" type="turtle_tf2_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle" type="string" value="turtle10" /> 
    </node>
    <node pkg="punto5" type="turtle_tf2_listener.py" 
          name="listener" output="screen"/>
    <node pkg="punto5" type="fixed_tf2_broadcaster.py"
          name="broadcaster_fixed" output="screen"/>
    <node pkg="punto5" type="fixed2_tf2_broadcaster.py"
          name="broadcaster_fixed2" output="screen"/>
    <node pkg="punto5" type="fixed3_tf2_broadcaster.py"
        name="broadcaster_fixed3" output="screen"/>
    <node pkg="punto5" type="fixed4_tf2_broadcaster.py"
        name="broadcaster_fixed4" output="screen"/>
    <node pkg="punto5" type="fixed5_tf2_broadcaster.py"
        name="broadcaster_fixed5" output="screen"/>
    <node pkg="punto5" type="fixed6_tf2_broadcaster.py"
        name="broadcaster_fixed6" output="screen"/>
    <node pkg="punto5" type="fixed7_tf2_broadcaster.py"
        name="broadcaster_fixed7" output="screen"/>
    <node pkg="punto5" type="fixed8_tf2_broadcaster.py"
        name="broadcaster_fixed8" output="screen"/>
    <node pkg="punto5" type="fixed9_tf2_broadcaster.py"
        name="broadcaster_fixed9" output="screen"/>

  </launch>
